test/gen-matter-4.test.js

```diff
@@ -113,7 +113,7 @@ test(
     },\\`)
 
     expect(ept).toContain(`#define GENERATED_ENDPOINT_TYPES { \\
-  { ZAP_CLUSTER_INDEX(0), 29, 377 }, \\
+  { ZAP_CLUSTER_INDEX(0), 29, 367 }, \\
```

Comment:
Why did this change again?

Reply:
see https://csamembers.slack.com/archives/C01B01GH64E/p1695906005160549